packages/sygma-ui/src/pages/TransferPage/TransferPage.tsx

```diff
@@ -249,7 +251,7 @@ const TransferPage = () => {
               disabled={!destinationChainConfig || formState.isSubmitting}
               name="receiver"
               label="Destination Address"
-              placeholder="0x · · · · · · · · · · · · ·"
+              placeholder="· · · · · · · · · · · · · ·"
```

Comment:
why this placeholder?

Reply:
I've changed this because of new substrate address format that is not started from `0x`